src/circuit/circuit_frame.rs

```diff
@@ -5722,13 +5778,88 @@ mod tests {
             )
             .unwrap();
 
-            popcount(
+            assert!(popcount(
                 &mut cs.namespace(|| format!("popcount {}", x.to_string())),
                 &bits,
                 &alloc_popcount.hash(),
-            );
+            )
+            .is_ok());
```

Comment:
Is this important to test? It doesn't seem to demonstrate anything about `popcount`'s correctness — just that there is no error returned.

Reply:
removed